jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -303,8 +309,25 @@ public Containerizer setAlwaysCacheBaseImage(boolean alwaysCacheBaseImage) {
     return this;
   }
 
+  /**
+   * Sets mirrors for a base image registry. Jib will try its mirrors in the given order before
+   * finally trying the registry.
+   *
+   * @param registry base image registry for which mirrors are configured
+   * @param mirrors a list of mirrors, where each element is in the form of {@code host[:port]}
+   * @return this
+   */
+  public Containerizer withRegistryMirrors(String registry, List<String> mirrors) {
```

Comment:
Naming wise, isn't this more "addRegistryMirrors" rather than "withRegistryMirrors"?

Reply:
Ah, makes sense. "add" was ambiguous originally when not using `MultiMap`, but I realized `ListMultiMap.putAll()` is additive than replacing.